prime-dotnet-webapi/Models/Certification.cs

```diff
@@ -44,5 +44,9 @@ public class Certification : BaseAuditable, IEnrolleeNavigationProperty
 
         [JsonIgnore]
         public Practice Practice { get; set; }
+
+        [NotMapped]
+        [JsonIgnore]
+        public string FullLicenceNumber { get { return $"{College?.Prefix}-{LicenseNumber}"; } }
```

Comment:
I feel there are some inconsistencies with licence vs license in our code.

Reply:
I agree, I defaulted to the "correct" spelling of licence for Canada here without thinking. I feel like there are 3 options:

1. Change this to FullLicenseNumber to be consistent, if wrong
2. Change LicenseNumber to LicenceNumber on the Certification only
3. Change every instance of License to Licence (where grammatically appropriate)

This PR already has a fair bit of refactoring in it, so whats a little bit more? On the other hand, I would like to get this feature in, it's a high profile one. I'm leaning towards 2.

Reply:
I have done #1 for consistency